### DISCUSSION TITLE
Technical question, can the SSD titlebar be different from headerbar?

### DISCUSSION THREAD

**Discussion by Feichtmeier** [ubuntu/yaru#2808] [Ideas]
There is this "bug" where some server side decoration titlebars remain dark when you switch the applications to light. Couldn't this be a solution to our never ender inverted headerbar discussion? I mean, if it would be technically possible to make this?

![grafik](https://user-images.githubusercontent.com/15329494/116077566-eb1fcd80-a695-11eb-9d75-33e25317df61.png)

@clobrano @Jupi007 do you know if this would be technically possible?

**Comment by Jupi007** [comment]
Sorry if I didn't understand correctly: you ask if it is possible to have the inverted headerbar only on SSD ?
If yes, I could say maybe with a `window:not(.csd) headerbar` selector.

**Reply by Feichtmeier** [reply]
yeah if this is possible :) 
I don't say this is _the_ solution :) Maybe it isn't even possible
Because I agree that the dark top rounds up the window well. Then we could even dream about going back to two themes (if if if if)

**Reply by Jupi007** [reply]
I'll try to see if it's possible so :)
If yes, I'll make a PR.

**Reply by Jupi007** [reply]
I did it: #2809

**Reply by Feichtmeier** [reply]
woah you are a wizard!!!
